DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: fec16
 Title: Data Package for the 2016 United States Federal Elections
-Version: 0.1.2.9000
+Version: 0.1.3
```

Comment:
Can you change this to 0.1.2.9001? We'll go to 0.1.3 once we're ready to resubmit to CRAN.